src/taxinyc/data_processor.py

```
@@ -0,0 +1,49 @@
+        self.X = None
+        self.y = None
+        self.preprocessor = None
+    
+    def load_data(self, table, sparksession):
+        return sparksession.read.table(table).toPandas()
+
+    def preprocess_data(self):
+        # Remove rows with missing target
+        target = self.config['target']
+        self.df = self.df.dropna(subset=[target])
+        
```

Ruff

Blank line contains whitespace

Remove whitespace from blank line

(W293)